Keas.Mvc/ClientApp/components/Access/AccessContainer.tsx

```diff
@@ -148,6 +156,10 @@ export default class AccessContainer extends React.Component<IProps, IState> {
             access: [...this.state.access, access]
         });
     }
+    if(!!this.props.accessAssigned)
```

Comment:
@laholstege No need for the `!!` here since this is a function and doesn't need the coalescence.  Just a regular `if(this.props.accessAssigned)` will serve the same purpose